### PR TITLE
Linter (Code quality)

### PR DESCRIPTION
**Pull request by ralf-meyer**
This PR addresses Issue #48. I know that enforcing a coding style may seem pedantic but the process has helped to uncover several bugs.

Please do not merge without proper code review! (And possibly testing of use cases not covered by the test-suite)

Most changes are minor, and I stopped lintering the directories Informatics, job_manager, and molscontrol at some point since it was not my intention to force a coding style on @chenruduan and @adityanandy. 

The most important part is probably the removal of bare `except` statements. This uncovered some smaller bugs that were previously caught by these statements. At certain points I just could not figure out what the expected Exception could be. For these cases I just used AssertionError as placeholder. Maybe someone can help to shed some light on these cases.

### CODE REVIEW

**Review by ralf-meyer — COMMENTED** [review]
deleted

File: molSimplify/Scripts/cellbuilder.py [hunk 1]
```diff
@@ -1148,7 +1148,7 @@ def slab_module_supervisor(args, rootdir):
         except:
             emsg.append('Unable to import unit cell at  ' +
                         str(args.unit_cell))
-            import_success == False
+            import_success = False
```
Comment (ralf-meyer) [inline molSimplify/Scripts/cellbuilder.py:1151]: Great example of the kind of bugs caught by lintering

File: molSimplify/Scripts/qcgen.py [hunk 2]
```diff
@@ -1082,7 +1082,7 @@ def molcgen(args, strfiles, method):
             mdir = rdir+'/'+nametrunc+mmd
             if not os.path.exists(mdir):
                 try:
-                    os.mkdirs(mdir)
+                    os.makedirs(mdir)
```
Comment (ralf-meyer) [inline molSimplify/Scripts/qcgen.py:1081]: Unsure if `mkdir()` or `makedirs()`. `mkdirs()` raised AttributeError that was caught by one of the bare except statements.

**Review by naveenarun — APPROVED** [review]
Looks good

File: molSimplify/Scripts/cellbuilder_tools.py [hunk 3]
```diff
@@ -370,14 +368,14 @@ def periodic_2d_distance(R1, R2, cell_vector):
         for v1shifts in [-1, 0, 1]:
             for yshifts in [-1, 0, 0]:
                 pass
-    d1 = sqrt(numpy.power(dim[0] - dx, 2)
-              + numpy.power(dim[1] - dy, 2)
+    d1 = sqrt(numpy.power(dim[0] - dx, 2)  # noqa: F821 (under construction)
```
[the remark this answers is not in the document]
Comment (ralf-meyer) [reply molSimplify/Scripts/cellbuilder_tools.py:371]: '"dim" is not defined'

**Pull request merged**